pkg/tlsf/contrib/include/tlsf-malloc.h

```diff
@@ -43,6 +43,28 @@
 extern "C" {
 #endif
 
+/**
+ * @brief Struct to hold the total sizes of free and used blocks
+ * Used for @ref tlsf_size_walker()
+ */
+typedef struct {
+    unsigned free;          /**< total free size */
+    unsigned used;          /**< total used size */
+} tlsf_size_container_t;
+
+/**
+ * Walk the memory pool to print all block sizes and to calculate
+ * the total amount of free and used block sizes.
+ *
+ * @warning This function is passed to tlsf_walk_pool()
+ *
+ * @param   ptr        Pointer to the current block.
+ * @param   size       Size of the current block at @p ptr.
+ * @param   used       Shows whether the current block is used or free.
+ * @param   user       Custom data expected to be of type ``pointer to @ref tlsf_size_container_t``
```

Comment:
You want the 

```
``...``
```

part to be typewriter font, right? As far as I can judge you can remove the `@ref` then, otherwise it shows up in the rendered doc ;-)